projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -590,6 +604,13 @@ public static function validate_task( $task ) {
 			return false;
 		}
 
+		$has_both_properties_for_counting = ( isset( $task['minimum_repetition_to_complete'] ) || isset( $task['repetition_count_callback'] ) ) && ! ( isset( $task['minimum_repetition_to_complete'] ) && isset( $task['repetition_count_callback'] ) );
+
+		if ( $has_both_properties_for_counting ) {
```

Comment:
Shouldn't this be `if ( ! $has_both_properties_for_counting ) {`?

Reply:
Thanks for catching it, the condition is correct but it was poorly named. I renamed it to: `missing_repepetition_property`, do you think it is clear now?

Reply:
yes, the other name was confusing. Thanks ❤️